### ISSUE TITLE
FR: v-if on the choose beer panel

### ISSUE THREAD

**Issue by billythekid** [sunscreem/dogpubtimemachine#41]
suggest not showing this panel at all unless a beer has been chosen, rather than just (re-)labelling to "choose a beer"

**Comment by sunscreem** [comment]
Agreed. I'm super keen to find a much better UI for this whole thing. You've given me much to think about. Hoping that a UI person from the forum chances across this and can chuck some mockups at me.
